site-content/source/modules/ROOT/pages/ecosystem.adoc

```diff
@@ -67,6 +67,7 @@ https://galaxy.ansible.com/community/cassandra[Ansible Cassandra Collection,wind
 
 https://ignite.apache.org/[Apache Ignite,window=blank]: Use Ignite as a traditional SQL database by leveraging JDBC drivers, ODBC drivers, or the native SQL APIs that are available for Java, C#, C++, Python, and more.
 
+https://github.com/aiven/astacus[Astacus,window=blank]: Astacus is a clustered database backup system that is meant to work with multiple open-source cluster databases, such as M3 and Apache Cassandra.
```

Comment:
will this render correctly without the blank line? (idk)

can you generate the website (`./run.sh website build`) and take a screenshot? 
that's the easiest way to speed up the review+test phase…